queue/bytes_queue.go

```diff
@@ -4,6 +4,8 @@ import (
 	"encoding/binary"
 	"log"
 	"time"
+
+	"golang.org/x/sys/unix"
```

Comment:
I like that bigcache has no external deps
Also with unix we should add a build tags to not use it on Windows

Reply:
fixed on #397